src/regressionmodel.jl

```diff
@@ -46,7 +46,7 @@ function modelmatrix end
 Return `X'X` where `X` is the model matrix of `model`.
 This function will return a pre-computed matrix stored in `model` if possible.
 """
-crossmodelmatrix(model::RegressionModel) = (x = modelmatrix(model); Symmetric(x' * x))
+crossmodelmatrix(model::RegressionModel; kwarg...) = (x = modelmatrix(model; kwarg...); Symmetric(x' * x))
```

Comment:
Better only allow `weighted::Bool=false`, and document it. Same for `modelmatrix`, and probably `momentmatrix` too?

Reply:
I think `modelmatrix` should explicitly have a `weighted::Bool=false` keyword argument.

Reply:
Also note that you'll have to do something like `weighted ? modelmatrix(model, weighted=true) : modelmatrix(model)` to avoid passing `weighted` when it's not necessary, which would break for packages that don't support it yet.

Reply:
Should be ok now